### ISSUE TITLE
Some logos on ontology pages are broken: GO, SO [because of https]

### ISSUE THREAD

**Issue by cmungall** [OBOFoundry/OBOFoundry.github.io#1990]
https://obofoundry.org/ontology/go.html
https://obofoundry.org/ontology/so.html

I think the unifying factor here is that these logos are served from http not https, does this make sense @kltm?

**Comment by kltm** [comment]
Works for me, I think?
That said, there are a lot of issues around mixed-schema web pages (https://support.mozilla.org/en-US/kb/mixed-content-blocking-firefox#w_what-is-mixed-content-and-what-are-the-risks) and different browsers are differently picky about this stuff.

**Comment by matentzn** [comment]
My Chrome does not show these logos!

**Comment by nlharris** [comment]
Those logos are broken for me in Chrome on Mac. @kltm not sure what you're suggesting as the fix.

**Comment by kltm** [comment]
While it depends on the browser, the best overall way to fix would be to remove cross-schema issues and host from https.

**Comment by nlharris** [comment]
Hmm. https://github.com/OBOFoundry/OBOFoundry.github.io/edit/master/ontology/go.md has
> depicted_by: http://geneontology.org/assets/go-logo-icon.mini.png

http://geneontology.org/assets/go-logo-icon.mini.png works, but https://geneontology.org/assets/go-logo-icon.mini.png does not.

Maybe I'm misunderstanding what you mean by "host from".

**Comment by kltm** [comment]
For GO, we do not currently offer HTTPS from that location. Until we get more HTTPS in place (https://github.com/geneontology/go-site/issues/53#issuecomment-817047884), it would have to be hosted elsewhere.

**Comment by nlharris** [comment]
I just ran into this broken-icon thing again and remembered that we already had a ticket for it. (Ok, I'm lying, I didn't remember, but I searched before starting to create a new ticket.)

Maybe we should just not attempt to show icons on ontology pages until this is fixed? Because it looks ugly, and makes the ontologies unfairly look abandoned.
<img width="496" alt="Screen Shot 2022-12-12 at 1 34 11 PM" src="https://user-images.githubusercontent.com/348704/207160814-bd8db68e-47a5-428e-83a4-63edf3970494.png">

**Comment by matentzn** [comment]
@erik-whiting can you:

- [ ] Move GO and SO logos to https://github.com/OBOFoundry/OBOFoundry.github.io/tree/master/images
- [ ] Change the GO and SO metadata to link to these images like we do for CL

**Comment by nlharris** [comment]
I can do this if @erik-whiting doesn't have time.

**Comment by erik-whiting** [comment]
I can do it, but can someone point me to where the logos currently are? should I grab them from their respective websites?

**Comment by erik-whiting** [comment]
here you go! https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2247

**Comment by erik-whiting** [comment]
Should we leave this open in case more logos start not showing up or should we go ahead and close this?

**Comment by matentzn** [comment]
No just close they will open a new one
